### PR TITLE
Solving Issue #202

### PR DESCRIPTION
**Pull request by j0rt3g4**
1 Important Note
1 Note related to hybrid configurations
1 Tip related to Old migrations from exchange 2010 to ExO.

Solving Issue #202
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ This topic shows you how to get public folders configured and running in a new o
 > [!NOTE]
 > For more information about the storage quotas and limits for public folders, see [Exchange Online Limits](https://go.microsoft.com/fwlink/p/?LinkID=391188).
 
+> [!IMPORTANT]
+> The procedure in this article guided you through the process of creating Public Folders the 1st time.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md]:
```suggestion
> The procedure in this article guides you through the process of creating public folders for the first time.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md [hunk 2]
```diff
@@ -86,3 +89,10 @@ If you want users to send mail to the public folder, you can mail-enable it. Thi
 The public folder will be mail-enabled and the name of the public folder will become the alias of the public folder. If you have multiple recipients with that name, the public folder's alias will be appended with a number. For example, if you have a distribution group named SalesTeam and you create a public folder named SalesTeam and then mail-enable it, the alias of that public folder will be SalesTeam1.
 
 For information about how to use Exchange Online PowerShell to mail-enable a public folder, see [Enable-MailPublicFolder](https://technet.microsoft.com/library/6fc7ba9a-62a8-4f41-811f-608363aa1397.aspx).
+
+> [!NOTE]
+> In case that you have a hybrid configuration, the Public folders created on Exchange Online are visible for only Mailboxes on Cloud. Oppositely, Public folders created on-Premises are visible for Mailboxes on the On-premises.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md]:
```suggestion
> If you have a hybrid configuration, the public folders created on Exchange Online are only visible to cloud-based mailboxes. Conversely, public folders created on-premises are only visible to on-premises mailboxes.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md [hunk 3]
```diff
@@ -86,3 +89,10 @@ If you want users to send mail to the public folder, you can mail-enable it. Thi
...
+
+> [!TIP]
+> In case that finished a migration from Exchange Server 2010 to Exchange Online with public folders, follow the following article:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md]:
```suggestion
> To complete a migration from Exchange Server 2010 to Exchange Online with public folders, see [Configure legacy on-premises public folders for a hybrid deployment](https://docs.microsoft.com/exchange/collaboration-exo/public-folders/set-up-legacy-hybrid-public-folders).
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md [hunk 4]
```diff
@@ -86,3 +89,10 @@ If you want users to send mail to the public folder, you can mail-enable it. Thi
...
+> [Configure legacy on-premises public folders for a hybrid deployment](https://docs.microsoft.com/exchange/collaboration-exo/public-folders/set-up-legacy-hybrid-public-folders)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md]:
```suggestion
```

**Comment by j0rt3g4** [comment]
Thank you @JohanFreelancer9. Seems correct.

**Pull request merged**